Uzduotis17/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+
+namespace Uzduotis17
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            /* Liepkite vartotojui įvesti teigiamą skaičių. 
+             * Ciklo pagalba šį skaičių išvesti mažėjimo tvarka iki kol jis pasieks 0.
+             * Kiekvieną kartą išvesti skirtingose eilutėse.
+            */
+
+            Console.Write("Please enter a positive number: ");
+            int number4 = int.Parse(Console.ReadLine());
+
+            if (number4 <= 0)
+            {
+                Console.WriteLine("Please enter a positive number.");
+            }
+            else
+            {
+                while (number4 > 0)
+                {
+                    Console.WriteLine(number4);
+                    number4--;
+
+                }
```

Comment:
Labai sudėtingas užrašytas sprendimas. Toks reikšmės patikrinimas su `if` nelabai prasmingas, nes vartotojui pasakoma, kad jis įvedė netinkamą reikšmę bet nesuteikiama galimybė vesti iš naujo.

Reply:
Be to, naudojant `while` ciklą jūs keičiate patį skaičių kurį įvedė vartotojas taigi po to netenkate duomenų (nebeturite to ką įvedė vartotojas)

Reply:
Atsiprašau, šioms užduotims kaip ir reikėjo naudoti `while` ciklą...